packages/neos-ui/src/Containers/Root.js

```diff
@@ -1,30 +1,34 @@
-import React from 'react';
+import React, {Component} from 'react';
 import PropTypes from 'prop-types';
 import {Provider} from 'react-redux';
 import {withDragDropContext} from '@neos-project/neos-ui-decorators';
 import style from './style.css';
 
 import Neos from './Neos/index';
 
-const Root = ({store, globalRegistry, configuration, menu, routes}) => {
-    const containerRegistry = globalRegistry.get('containers');
+class Root extends Component {
```

Comment:
Can we use `PureComponent` here instead of `Component`?

Reply:
Sure